src/ScrollingBookView.ts

```diff
@@ -11,19 +11,23 @@ export default class ScrollingBookView implements BookView {
         this.iframe = iframe;
     }
 
-    public setTopMargin(topMargin: number): void {
-        this.topMargin = topMargin;
+    public setSideMargin(margin: number) {
+        this.sideMargin = margin;
```

Comment:
I think I did it that way because I wanted to have a setter with no getter, but I like your way better. And I didn't know about TypeScript's get and set syntax.